### ISSUE TITLE
Zero index label

### ISSUE THREAD

**Issue by iSWORD** [koss-shtukert/laravel-nova-select2-auto-complete#40]
There seems to be an issue when showing the label for the option at index 0 when storing the selected value in an `integer` database field.

Example:
```php
public function fields(Request $request)
{
    return [
        Select2::make('Type')->options(['Foo', 'Bar', 'Milk']),
        //
    ];
}
```

Where type is defined as the following in the migration:

```php
public function up()
{
    Schema::create('peanuts', function (Blueprint $table) {
        $table->integer('type')->default(0);
        //
    });
}
```

When selecting item at index 0 "Foo" it fails to show its label on the show, index or edit views.

Thank you,

**Comment by koss-shtukert** [comment]
Hello, @iSWORD I'll check and fix this issue as soon as possible, thanks!

**Comment by koss-shtukert** [comment]
Fixed on Release [v1.21](https://github.com/koss-shtukert/laravel-nova-select2-auto-complete/releases/tag/v1.21)
